lib/MwbExporter/Formatter/Doctrine2/Model/Column.php

```diff
@@ -135,6 +135,21 @@ public function isUuid($defaultValue = false)
         return false;
     }
 
+    public function extractDoctrineType(): ?string
+    {
+        $comment = $this->parameters->get('comment');
+        $type = null;
+        $matches = null;
+
+        if (preg_match("{^\(DC2Type:[ ]*([a-z_]+)[ ]*\)$}", $comment, $matches)) {
```

Comment:
```suggestion
        if (preg_match("{^\(DC2Type:[ ]*([A-Za-z_]+)[ ]*\)$}", $comment, $matches)) {
```

Not sure if this is required but as far as I can see preg_match is case-sensitive - unless maybe all types are required to be lowercase?